ui_framework/test/common_html_props.js

```diff
@@ -0,0 +1,5 @@
+export const commonHtmlProps = {
```

Comment:
Maybe it would be even better to rename these `requiredSupportProps` or something. The current name implied to me that this was some sort of shortcut for not having to come up with random attributes to pass into components. I was planning to ask that they just be included into each test file, rather than extracting them into a shared module, but since they are more about ensuring that each component support them this makes more sense.